src/main/scala/ai/privado/passes/SQLParser.scala

```diff
@@ -27,6 +27,7 @@ import ai.privado.cache.RuleCache
 import ai.privado.model.sql.{SQLColumn, SQLQuery}
 import ai.privado.tagger.PrivadoParallelCpgPass
 import ai.privado.utility.{SQLParser => UtilitySQLParser, Utilities}
+import ai.privado.utility.SQLNodeBuilder
```

Comment:
We already have a SQL utility file, let's place this code in that file, instead of creating a new one

Reply:
Added new changes in SQLParser: https://github.com/Privado-Inc/privado-core/pull/694/commits/b2c7217755a9d656a2d7ff62a73d56471270c784